### DISCUSSION TITLE
org.freedesktop.impl.portal.PermissionStore was not provided

### DISCUSSION THREAD

**Discussion by Casshew** [aclap-dev/video-downloadhelper#196] [Help and Q&A]
### Your question

Everything seems to go well until I get an error when I run 
`flatpak permission-set webextensions net.downloadhelper.coapp snap.firefox yes`
which returns 
`The name org.freedesktop.impl.portal.PermissionStore was not provided by any .service files`

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unneeded
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: no.

`{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/opt/vdhcoapp/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/cass",
  "converterBinary": "/opt/vdhcoapp/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360"
`

**Comment by paulrouget** [comment]
Which version of Firefox do you use? (linux distribution, flatpak, snap, firefox version)

**Reply by Casshew** [reply]
123.0 64bit
Firefox Snap for Ubuntu
canonical-002 - 1.0

**Reply by paulrouget** [reply]
I have no idea where this comes from. First time we encounter that error.

What Desktop environment do you use? Gnome? Default Ubuntu DE?

**Reply by Casshew** [reply]
I think it was much simpler than I thought, I did try a few other things before this so im not confident it is this, but once I logged out and logged back in the popup from firefox immediately came up
